Source/WebCore/dom/wpe/PointerEventWPE.cpp

```diff
@@ -72,10 +72,10 @@ Ref<PointerEvent> PointerEvent::create(const AtomString& type, const PlatformTou
 }
 
 PointerEvent::PointerEvent(const AtomString& type, const PlatformTouchEvent& event, const Vector<Ref<PointerEvent>>& coalescedEvents, const Vector<Ref<PointerEvent>>& predictedEvents, CanBubble canBubble, IsCancelable isCancelable, unsigned index, bool isPrimary, Ref<WindowProxy>&& view, const IntPoint& touchDelta)
```

Comment:
@lilyspiniolas  Thanks for working on this.

The argument type of touchDelta need to be changed to FloatPoint as well. 
Also type of touchDelta from the function definition of  PointerEvent::create from line number 57, 63, 69 needs to be changed to FloatPoint to fix the current compile error.